src/canister/individual_user_template/src/api/canister_lifecycle/post_upgrade.rs

```diff
@@ -63,3 +64,12 @@ fn save_upgrade_args_to_memory() {
         }
     });
 }
+
+fn migrate_excessive_tokens(){
+    CANISTER_DATA.with(|canister_data_ref_cell| {
+        let mut canister_data_ref_cell = canister_data_ref_cell.borrow_mut();
+        if canister_data_ref_cell.utility_token_balance > 18_00_00_00_00_00_00_00_00_00 {
```

Comment:
i think it is canister_data_ref_cell.my_token_balance.utility_token_balance

Reply:
done. my bad.